src/systems/ec/implementations/react/components/gallery/src/Gallery.jsx

```diff
@@ -32,6 +35,20 @@ export const Gallery = ({
 
       <Button label={viewAllLabel} variant="ghost" data-ecl-gallery-all />
 
+      <div className="ecl-gallery__footer">
+        <Link
+          href="#"
+          aria-label={footerLinkLabel}
+          label={footerLinkLabel}
+          icon={{
+            shape: 'ui--external',
+            size: 's',
+          }}
+          iconPosition="after"
+          className="ecl-link--standalone"
```

Comment:
here you should use the link variant "standalone", which will add the correct classes automatically